api/v1beta1/consts.go

```diff
@@ -166,3 +166,8 @@ const (
 	// value for the label is the CAPI Cluster Name.
 	OwnedByClusterLabelKey = NameAzureProviderPrefix + string(ResourceLifecycleOwned)
 )
+
+const (
+	// NetworkPluginName is the name of the network plugin.
+	NetworkPluginName = "azure"
```

Comment:
Should we also name this to distinguish "azure" from "kubenet" which is also a valid network plugin?